sigs.yaml

```diff
@@ -0,0 +1,83 @@
+sigs:
+- dir: sig-data-scientist-experience
+  name: Data Scientist Experience
+  mission_statement: >
+    Mission statement goes here.
+
+  charter_link: charter.md
+  label: data-scientist-experience
+  leadership:
+    chairs:
+    - github: abc
+      name: ABC
+      company: Red Hat
+    - github: XYZ
+      name: XYZ
+      company: Red Hat
+    tech_leads:
+    - github: abc
+      name: ABC
+      company: Red Hat
+    - github: XYZ
+      name: XYZ
+      company: Red Hat
+  meetings:
+  - description: Regular SIG Meeting
+    day: Wednesday
```

Comment:
The day and time are subject to change right?

Reply:
Correct, this is just a placeholder until the SIGs are formed.